packages/ui-map-components/src/components/Table/getMapTableData.tsx

```diff
@@ -13,22 +13,26 @@ const FirstColumnCell = styled.span`
 `;
 
 const processColumns = (serieses: Series[]) => {
+  console.log(serieses);
   if (!serieses) {
     return [];
   }
 
   const configColumns = serieses.map(column => {
-    return { accessor: column.key, Header: column.name };
+    return {
+      accessor: (row: any) => row[column.key],
+      Header: column.name,
+    };
   });
 
   return [
     {
       Header: 'Name',
       accessor: 'name',
       // eslint-disable-next-line react/prop-types
-      Cell: ({ value }: { value: string | number | boolean | undefined }) => (
-        <FirstColumnCell>{String(value)}</FirstColumnCell>
-      ),
+      Cell: (value: { value: string | number | boolean | undefined }) => {
```

Comment:
this should be just the same as before but it looks like there are a couple of small changes.